packages/subiquity_client/generator/generator.py

```diff
@@ -209,6 +209,10 @@ def __init__(self, node: ast.ClassDef):
     def visit_AnnAssign(self, node: ast.AnnAssign):
         self.properties.append(PropertyGenerator(node))
 
+    # skip function definitions within the class
+    def visit_FunctionDef(self, node: ast.FunctionDef):
+        pass
+
```

Comment:
This change is needed since the generator adds to `actions` properties to the dart class when parsing the types from [here](https://github.com/canonical/subiquity/blob/3dba9819c4b71e11f03250442c31821b5ee87ad3/subiquity/common/types/storage.py#L303-L307), where the nested class method contains a local variable also named `actions`